files/waypoints/update.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packageManager": "github",
+  "name": "waypoints",
+  "repo": "imakewebthings/waypoints",
+  "files": {
+  	"include": [
```

Comment:
All these files are in the lib/ directory. You need to use the "basePath" parameter to set it.

Also instead of listing all files you can simply do `"*.min.*"` and `./shortcuts/*.min.js`